src/next-reminders/lib/db.ts

```diff
@@ -1,6 +1,6 @@
-import { Pool, QueryConfig, QueryResult } from 'pg';
-import { runWithLogging } from '../../lib/db';
-import { getCurrentReminderPeriod } from './utils';
+import {Pool, QueryConfig, QueryResult} from 'pg';
+import {runWithLogging} from '../../lib/db';
+import {getCurrentReminderPeriod} from './utils';
```

Comment:
might want to undo this change?

Reply:
I'm not sure where this has come from, but it didn't break the build!

Reply:
Yeah looks like we don't run linting as part of the build. I can create a quick PR to fix it

Reply:
https://github.com/guardian/support-reminders/pull/29